tiledb/api/src/array/dimension/mod.rs

```diff
@@ -970,91 +525,28 @@ mod tests {
         }
     }
 
+    /// Test that the arbitrary dimension construction always succeeds
     #[test]
-    fn subarray_strategy_dense() {
```

Comment:
This test doesn't actually use any tiledb APIs so it now lives in `serde/src/array/dimension/strategy.rs`